### DISCUSSION TITLE
Spamming no found error when closing channel

### DISCUSSION THREAD

**Discussion by ghost** [geckosio/geckos.io#197] [Help / Questions] [answered]
Hey, when I use channel.close(), it works fine, but it start spamming this error on the console : 

![image](https://user-images.githubusercontent.com/86203108/165704698-ba2ce6dd-9ef3-43f3-91bd-e2f222b23490.png)

Is it my fault, or is there a bug ? If it's a bug, can you please fix it, it's annoying for testing

**Comment by yandeu** [comment, marked as answer]
If you close the channel within the first 10 seconds or so, the client can't fetch additional candidates. That's fine. Nothing to worry about.
